### PR TITLE
[DEC-2075] x/vest test coverage + ValidateBasic fixes for Msgs

### PR DESCRIPTION
**Pull request by clemire**

### CODE REVIEW

File: protocol/testutil/keeper/vest.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package keeper
```
Comment (clemire) [inline protocol/testutil/keeper/vest.go:1]: Added this file for cli, module tests

File: protocol/x/vest/client/cli/query_vest_entry_test.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package cli_test
```
Comment (clemire) [inline protocol/x/vest/client/cli/query_vest_entry_test.go:1]: test coverage for query_vest_entry.go

File: protocol/x/vest/types/tx.go [hunk 3]
```diff
@@ -21,6 +22,15 @@ func (msg *MsgSetVestEntry) GetSigners() []sdk.AccAddress {
 }
 
 func (msg *MsgSetVestEntry) ValidateBasic() error {
+	if _, err := sdk.AccAddressFromBech32(msg.Authority); err != nil {
```
Comment (clemire) [inline protocol/x/vest/types/tx.go:25]: Bug: ValidateBasic was missing authority check here - added with test coverage.

File: protocol/x/vest/types/tx.go [hunk 4]
```diff
@@ -37,6 +47,21 @@ func (msg *MsgDeleteVestEntry) GetSigners() []sdk.AccAddress {
 }
 
 func (msg *MsgDeleteVestEntry) ValidateBasic() error {
-	// TODO
+	if _, err := sdk.AccAddressFromBech32(msg.Authority); err != nil {
```
Comment (clemire) [inline protocol/x/vest/types/tx.go:50]: Unimplemented - fixed! With test coverage.

File: protocol/x/vest/types/vest_entry_test.go [hunk 5]
```diff
@@ -81,6 +81,28 @@ func TestValidate(t *testing.T) {
 			},
 			expectedErr: types.ErrInvalidStartAndEndTimes,
 		},
+		{
+			desc: "start_time not utc",
```
Comment (clemire) [inline protocol/x/vest/types/vest_entry_test.go:85]: missing coverage for invalid time zones

**Review by teddyding — APPROVED** [review]

**Review by teddyding — APPROVED** [review]

**Pull request merged**